lms/djangoapps/django_comment_client/forum/views.py

```diff
@@ -402,7 +404,9 @@ def single_thread(request, course_key, discussion_id, thread_id):
             'user_cohort': user_cohort,
             'sort_preference': cc_user.default_sort_key,
             'category_map': course_settings["category_map"],
-            'course_settings': json.dumps(course_settings)
+            'course_settings': json.dumps(course_settings),
+            'disable_courseware_js': True,
+            'uses_pattern_library': True,
```

Comment:
Is "uses_pattern_library" something you always have to specify? If there isn't already something, it would be splendid if you created a wiki page outlining how to convert an edx-platform page to the pattern library.